Formula/qt.rb

```diff
@@ -67,25 +63,15 @@ def install
       -no-rpath
       -pkg-config
       -dbus-runtime
+      -no-avx2
     ]
 
```

Comment:
According to https://www.qt.io/blog/qt-6-build-system, CMake now is the default build system.

In my fork, I use these argument to avoid the cmake link issue:

    args = std_cmake_args.reject { |s| s["CMAKE_INSTALL_PREFIX"] } + %W[
      -DCMAKE_INSTALL_PREFIX=/usr/local
      -DCMAKE_STAGING_PREFIX=#{prefix}

      -DICU_ROOT=#{Formula["icu4c"].opt_prefix}
      -DLLVM_INSTALL_DIR=#{Formula["llvm"].opt_prefix}

      -DINSTALL_ARCHDATADIR=share/qt
      -DINSTALL_DATADIR=share/qt

      -DINSTALL_DESCRIPTIONSDIR=share/qt/modules
      -DINSTALL_DOCDIR=share/doc/qt
      -DINSTALL_EXAMPLESDIR=share/qt/examples
      -DINSTALL_MKSPECSDIR=share/qt/mkspecs
      -DINSTALL_PLUGINSDIR=share/qt/plugins
      -DINSTALL_QMLDIR=share/qt/qml
      -DINSTALL_TESTSDIR=share/qt/tests

      -DFEATURE_appstore_compliant:BOOL=ON
      -DFEATURE_freetype:BOOL=ON
      -DFEATURE_pkg_config:BOOL=ON
      -DFEATURE_relocatable:BOOL=OFF
      -DFEATURE_zstd:BOOL=ON
    ]

    system "cmake", "-G", "Ninja", ".", *args
    system "ninja"
    system "ninja", "install"

Reply:
In addition, brew linkapps is deleted, I don't know the result after use this workaround:

    # FIXME: this step just moves `*.app` bundles into `libexec` and create link
    libexec.mkpath
    Pathname.glob("#{bin}/*.app") do |app|
      mv app, libexec
      bin.install_symlink "#{libexec/app.stem}.app/Contents/MacOS/#{app.stem}"
    end